components/automate-cli/pkg/verifyserver/constants/systemuserconstant.go

```diff
@@ -0,0 +1,23 @@
+package constants
+
+const (
+	USERNAME                                   = "hab"
+	GROUPNAME                                  = "hab"
+	USERADDCMD                                 = "useradd"
+	USERDELCMD                                 = "userdel"
+	SYSTEM_USER_HAB_VALIDATION_SUCCESS_TITLE   = "User creation/validation check"
+	SYSTEM_USER_HAB_SUCCESS_MSG                = "User is created or found successfully"
+	SYSTEM_GROUP_HAB_VALIDATION_SUCCESS_TITLE  = "Group creation/validation check"
+	SYSTEM_GROUP_HAB_SUCCESS_MSG               = "Group is created or found successfully"
+	SYSTEM_USERANDGROUP_MAPPING_SUCCESS_TITLE  = "User and group mapping successfully"
+	SYSTEM_USERANDGROUP_MAPPING_SUCCESS_MSG    = "User and group mapping successful"
+	SYSTEM_USER_HAB_VALIDATION_FAILURE_TITLE   = "User validation failure"
+	SYSTEM_USER_HAB_ERROR_MSG                  = "User found but failed the validation"
+	SYSTEM_USER_HAB_RESOLUTION_MSG             = "Check the user name"
+	SYSTEM_GROUP_HAB_VALIDATION_FAILURE_TITLE  = "Group Validation failure"
+	SYSTEM_GROUP_HAB_ERROR_MSG                 = "Group found but failed the validation"
```

Comment:
This message needs to be changed to showcase that hab group was not found

Reply:
Done